src/Components/Publishing/Sections/ImageSetPreview/ImageSetPreview.tsx

```diff
@@ -104,6 +106,8 @@ export const Img = styled.img`
   width: 100%;
 `
 
+export const ImgContainer = styled.div``
```

Comment:
Do we need this?

Reply:
I needed to export this so I could hook into it from another component, open to suggestions for better ways to accomplish.

Reply:
Ah makes sense then. I'm sure there's a better way but let's stick with this for now.